src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/TelemetryService.kt

```diff
@@ -70,8 +66,10 @@ class TelemetryService(private val telemetryClient: TelemetryClient) {
   }
 
   fun trackEvent(eventType: TelemetryEventType, customDimensions: Map<EventKeys, String?>) {
-    log.debug("Sending telemetry event ${eventType.eventName} ")
-    val transformedDimensions: Map<String, String?> = customDimensions.entries.associate { it.key.name to it.value }
-    telemetryClient.trackEvent(eventType.eventName, transformedDimensions, null)
+    val correlationId = telemetryClient.context.operation.id
+
+    val updatedDimensions = customDimensions.entries.associate { it.key.name to it.value } + mapOf("CORRELATION_ID" to correlationId)
```

Comment:
Is there any point in adding this correlation ID if it is already present as the operationId?

Reply:
This whole change doesn't seem to add any value - we are just logging this value twice instead of once as far as I can see

Reply:
I agree this doesn't need to be added to the exisiting telemetry. However, it does need pulling out so we can add it it to the audit / event store.